django/contrib/admin/options.py

```diff
@@ -909,11 +909,31 @@ def construct_change_message(self, request, form, formsets, add=False):
         """
         Construct a change message from a changed object.
         """
+        def convert_changed_list(obj, changed_data):
+            """ Add verbose names of models to changed data """
+            cls = type(obj)
```

Comment:
I am not sure if using `cls` as the variable name is best practice. This has been maintained from the original patch, I would personally have used `form_class`. Is there a preference for any / third option?